VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+# Validators FAQ
+
+## General concepts
+
+### What is a validator?
+
+The Cosmos Hub is based on Tendermint, which relies on a set of validators to secure the network. The role of validators is to run a full-node and participate in consensus by broadcasting votes which contain cryptographic signatures signed by their private key. Validators commit new blocks in the blockchain and receive rewards in exchange for their work. They must also participate in governance by voting on proposals.
+
+### What is 'staking'?
+
+The Cosmos Hub is a public proof-of-stake blockchain, meaning that validator's weight is determined by the amount of staking tokens (Atoms) bonded as collateral. These Atoms can be staked directly by the validator or delegated to them by Atom holders.
+The weight of a node determines wether or not it is a validator, but also how frequently this node will have to propose a block and how much reward it will obtain. Initially, only the top 100 validator candidates with the most weight will be validators. If validators double sign, are frequently offline or do not participate in governance, their staked ATOMs (including ATOMs of users that delegated to them) can be slashed.
+
+### What is a full-node?
+
+A full-node is a program that fully validates transactions and blocks of a blockchain. It is distinct from a light-node that only processes block headers and a small subset of transactions. Running a full-node requires more resources than a light-node but is necessary in order to be a validator. In practice, running a full-node only implies running a non-compromised and up-to-date version of the software with low network latency and without down time. 
+
+### What is a delegator?
+
+Delegators are Atoms holders who cannot, or do not want to, run validator operations themselves. Through the Cosmos UI, a user can delegate Atoms to a validator and obtain a part of its reward in exchange.
```

Comment:
`... are Atom holders ...`

`... do not want to run validator ...`

Reply:
ok